.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release X-Ray Go SDK
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.3.0
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v2
+
+      - name: Initialize Github name and email
```

Comment:
Ah - I dont think this is needed to create a tag (it would be to create a commit)

Reply:
Good catch! Yeah I will remove it.